cli/azd/pkg/templates/source_manager.go

```diff
@@ -96,11 +96,13 @@ func (sm *sourceManager) List(ctx context.Context) ([]*SourceConfig, error) {
 			sourceConfig.Key = key
 			sourceConfigs = append(sourceConfigs, sourceConfig)
 		}
-	}
-
-	// If not sources have been registered, add the default source.
-	if len(sourceConfigs) == 0 {
-		sourceConfigs = append(sourceConfigs, SourceDefault)
+	} else {
```

Comment:
Does `default` now not mean the "default source'?

Reply:
Correct - default is more like a **legacy** template source now.  @savannahostrowski any thoughts on how we would want to reference this if users wanted to go back to our manually curated list?

Reply:
Hmmm, that's an interesting question. I could see us still having them labelled as default. I don't see a name that's better suited off the top of my head.